landoapi/workers/landing_worker.py

```diff
@@ -416,12 +418,14 @@ def run_job(
                     f"`Temporary error ({e.__class__}) "
                     f"encountered while pushing to {repo_push_info}"
                 )
+                logger.exception(message)
```

Comment:
nit: I'm not sure we need the full stack trace for all of these errors, since all except `HgmoInternalServerError` are very specific. Might make more sense to move `except HgmoInternalServerError` to its own line instead.

Reply:
I think having more info in the logs as a default makes sense, we can always filter that out if the full stack trace is too much context. It would be worse to have an incident in progress and have to re-deploy Lando to add more logging so we could fix the problem. :)